### PR TITLE
Package pilat.1.6

### PR DESCRIPTION
**Pull request by Stevendeo**
### `pilat.1.6`
Polynomial invariant generator

---
* Homepage: https://github.com/Stevendeo/Pilat/
* Source repo: git://github.com/Stevendeo/Pilat.git
* Bug tracker: https://github.com/Stevendeo/Pilat/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/pilat/pilat.1.6/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+build: ["sh" "make.sh"]
+install: ["sh" "install.sh"]
+depends: [
+    "ocaml" {>= "4.08.1"}
+    "ocamlfind"
+    "lacaml" {<= "11.0.8"}
+    "zarith" {<= "1.12"}
+    "why3" {<= "1.3.0"}
+    "frama-c" {>= "22.0" <= "25.0"}
+    "conf-gtksourceview"
```
Comment (kit-ty-kate) [inline packages/pilat/pilat.1.6/opam:17]:
```suggestion
    "lacaml" {>= "11.0.8"}
    "zarith" {>= "1.12"}
    "why3" {>= "1.3.0"}
    "frama-c" {>= "22.0"}
```

Comment (Stevendeo) [reply packages/pilat/pilat.1.6/opam]: The next version of frama-c may break the plug-in and force me to update the version's upper bound. Is it a good practice not to have upper bounds in general?

**Comment by mseri** [comment]
The lower bound issue comes from why3:
```
#=== ERROR while compiling why3.1.3.3 =========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.08/.opam-switch/build/why3.1.3.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build make -j31 all opt byte
# exit-code            2
# env-file             ~/.opam/log/why3-367-06e1a4.env
# output-file          ~/.opam/log/why3-367-06e1a4.out
### output ###
[...]
# File "src/parser/report.ml", line 34, characters 2-22:
# 34 |   current_state_number (env checkpoint)
#        ^^^^^^^^^^^^^^^^^^^^
# Error: Unbound value current_state_number
# make: *** [Makefile:2076: src/parser/report.cmo] Error 2
# make: *** Waiting for unfinished jobs....
```
I don't know what is needed to fix it

**Comment by kit-ty-kate** [comment]
ok let’s merge for now then. Thanks!

**Pull request merged**